database/migrations/16_updateVersionsUuidInEnvironmentsTable.go

```diff
@@ -6,7 +6,7 @@ import (
 
 func updateVersionsUUIDInEnvironmentsTable() *migrate.Migration {
 	return &migrate.Migration{
-		Id: "15",
+		Id: "16",
```

Comment:
It was recorded in gorp but will be removed. These migrations are updating values not adding/removing columns or constraints. So removing the entries in gorp will allow them to re-run and properly update the values in the correct order and places.